Plugins/Public/base_plugin/Main.cpp

```diff
@@ -2757,3 +2784,36 @@ EXPORT PLUGIN_INFO* Get_PluginInfo()
 	p_PI->lstHooks.push_back(PLUGIN_HOOKINFO((FARPROC*)&Plugin_Communication_CallBack, PLUGIN_Plugin_Communication, 11));
 	return p_PI;
 }
+
+void ResetShieldStrength() {
```

Comment:
Should this be named ResetAllBasesShieldStrength or something?

Reply:
Good catch, updated to avoid naming overlap.